docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
+   ```
 
-### Install Teleport on your Node
+1. Copy the token and assign it to an environment variable on the computer you are enrolling 
+as a resource:
+   
+   ```code
+   $ export INVITE_TOKEN=<token>
+   ```
 
-On the host where you will run your Teleport Node, follow the instructions for
-your environment to install Teleport.
+## Step 2/3. Apply a static label
 
-(!docs/pages/includes/install-linux.mdx!)
-
-### Generate a token
-
-On your local machine, use the `tctl` tool to generate an invite token that your
-Node will use to join the cluster. In the following example, a new token is
-created with a TTL of five minutes:
-
-```code
-# Generate a short-lived invitation token for a new node:
-$ tctl tokens add --ttl=5m --type=node
-# The invite token: (=presets.tokens.first=)
-
-# You can also list all generated non-expired tokens:
-$ tctl tokens ls
-# Token                            Type            Expiry Time
-# ------------------------         -----------     ---------------
-# (=presets.tokens.first=)         Node            25 Sep 18 00:21 UTC
-```
-
-Copy the token. On the remote host that you will add to your Teleport cluster as a
-Node, assign the token to an environment variable:
-
-```code
-$ export INVITE_TOKEN=<token>
-```
-
-We will use this in the next step.
-
-## Step 2/3. Apply a static label to your Node
-
-You can configure Node labels by editing Teleport's configuration file, then
+You can configure static labels by editing the Teleport configuration file, then
 starting Teleport.
 
-On the host where you will run your Node, paste the following content into
-`/etc/teleport.yaml` and adjust it for your environment:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+To add a static label:
+
+1. Open the Teleport configuration file—by default, `/etc/teleport.yaml`—in an editor on the computer 
+where you installed the Teleport agent.
+
+1. Locate the `labels` configuration under the `ssh_service` section.
+
+1. Add the static label key and value. 
+   For example, add `environment` as the label key and `dev` as the value:
+
+   ```yaml
+   ssh_service:
+     enabled: true
+     labels:
+       environment: dev
+   ```
+
+1. Save your changes and close the file.
+
+1. Start Teleport with the invitation token you saved in the `INVITE_TOKEN` environment variable:
+   
+   ```code
+   $ sudo teleport start --token=${INVITE_TOKEN?}
+   ```
+
+1. Verify that you have added the label by running the following command 
+on your local computer. 
+      
+   ```code
+   $ tsh ls --query 'labels["environment"]=="dev"'
+   ```
+
+   You should see output similar to the following:
+
+   ```code
+   Node Name        Address    Labels                                                                                                       
+   ---------------- ---------- ------------------------------------------------------------------------
+   ip-192-168-13-57 ⟵ Tunnel   environment=dev,hostname=ip-192-168-13-57,teleport.internal/resource-id
```

Comment:
We just landed a change that would hide the teleport.internal/ label. Probably makes sense to remove it here for consistency and to avoid confusing the reader.